repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/model/CaseEventField.java

```diff
@@ -153,6 +154,19 @@ public void setPublish(Boolean publish) {
         this.publish = publish;
     }
 
+    /**
+     * whenever we want to publish this event field.
+     **/
+    @ApiModelProperty(value = "allows services to define the default value of the CaseAccessCategory")
```

Comment:
```suggestion
    @ApiModelProperty(value = "allows services to define the default value of top level case fields")
```
`CaseAccessCategory` is the first usage, but in theory can be used for anything

Reply:
Fixed in [5e6bcf3](https://github.com/hmcts/ccd-definition-store-api/pull/1111/commits/5e6bcf3530b16e5bff37a8f0c02855a0d686d834)